app/routes.py

```diff
@@ -1,2 +1,76 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
 
+bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+# VALIDATE PLANET HELPER FUNCTIONS
+def validate_model(cls, model_id):
```

Comment:
Even though there's only one model in this codebase, this function is generic and could be used with any model, so it would be ideal to move it into its own helpers file.